app/scripts/models/reliers/oauth.js

```diff
@@ -17,7 +17,7 @@ define([
   'lib/relier-keys',
   'lib/url'
 ], function (_, Relier, ResumeToken, p, OAuthErrors, RelierKeys, Url) {
-  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state'];
+  var RELIER_FIELDS_IN_RESUME_TOKEN = ['state', 'keys'];
```

Comment:
Adding `keys` here doesn't do anything, removing **functional tests** still pass. Currently we are not parsing the resume token anywhere AFAIK. I'm adding parsing in https://github.com/mozilla/fxa-content-server/pull/2366/files#diff-ca4d7096469c2a76694bb8214760cbaaR104

Not sure if we want to keep this here. Is this just for a the "different browser" flow to know that the relier wanted keys?